src/Compilers/Core/Portable/InternalUtilities/ReadOnlyHashSet.cs

```diff
@@ -0,0 +1,22 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+
+namespace Roslyn.Utilities
+{
+    internal struct ReadOnlyHashSet<T> : IReadOnlySet<T>
```

Comment:
If one of them is not used why not just delete it? :)

---
In reply to: [74471878](https://github.com/dotnet/roslyn/pull/12353#discussion_r74471878) [](ancestors = 74471878)